### PR TITLE
[SQL] using variable instead of subselect

### PR DESCRIPTION
**Pull request by xlecours**
## Brief summary of changes
Adjust the SQL patch for mysql server error:
`ERROR 1093 (HY000): You can't specify target table 'LorisMenu' for update in FROM clause`

#### Testing instructions (if applicable)
run the patch. There should be no error.

### CODE REVIEW

**Comment by driusan** [comment]
What version of MySQL/MariaDB should this be tested on?

File: SQL/New_patches/2019-11-12-Rename_modules_QC_and_DTH.sql [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
-UPDATE LorisMenu SET Label='Imaging Quality Control', Link='imaging_qc/', Parent=(SELECT ID FROM LorisMenu WHERE Label='Imaging'), OrderNumber=(SELECT MAX(OrderNumber)+1 FROM LorisMenu WHERE Parent=(SELECT ID FROM LorisMenu WHERE Label='Imaging')) WHERE Link='quality_control/';
+SELECT MAX(OrderNumber)+1 INTO @ordernumber FROM LorisMenu WHERE Parent=(SELECT ID FROM LorisMenu WHERE Label='Clinical');
+SELECT ID INTO @parentid FROM LorisMenu WHERE Label='Clinical';
+
+UPDATE LorisMenu SET Label='Behavioural Quality Control', Link='behavioural_qc/', Parent=@parentid, OrderNumber=@ordernumber  WHERE Link='data_team_helper/';
+
+SELECT MAX(OrderNumber)+1 INTO @ordernumber FROM LorisMenu WHERE Parent=(SELECT ID FROM LorisMenu WHERE Label='Imaging');
+SELECT ID INTO @parentid FROM LorisMenu WHERE Label='Imaging';
+UPDATE LorisMenu SET Label='Imaging Quality Control', Link='imaging_qc/', Parent=@parentid, OrderNumber=@ordernumebr WHERE Link='quality_control/';
```
Comment (ridz1208) [inline SQL/New_patches/2019-11-12-Rename_modules_QC_and_DTH.sql]:
```suggestion
UPDATE LorisMenu SET Label='Imaging Quality Control', Link='imaging_qc/', Parent=@parentid, OrderNumber=@ordernumber WHERE Link='quality_control/';
```

**Comment by ridz1208** [comment]
@driusan jst tested on mariadb 10.3, it works after fixing the typo

**Comment by driusan** [comment]
@ridz1208 can you update your review?

**Pull request merged**